faq/pophelp/prefs/prefs_pophelp.inc

```diff
@@ -116,8 +116,8 @@ so be sure that your current email address is on record.</p>")
     'content' =>
 _("<p>This setting controls who is able to see your statistics.</p>
 <ul>
-<li><b>Private:</b> Only registered users.</li>
-<li><b>Anonymous:</b> Only you.</li>
+<li><b>Private:</b> Only logged-in registered users.</li>
```

Comment:
`registered` seems redundant here -- they can't be logged in if they're not registered.

Reply:
Good point -- and we switched from `registered` to `logged in` elsewhere. I'll get that change in when I get the charsuite one done.

Reply:
Done.